apps/frontend/utils/aggregate/firstAgg.ts

```diff
@@ -0,0 +1,10 @@
+import { compact } from 'lodash';
+
+export const firstAgg = (
```

Comment:
I feel like it could make more sense to have all these functions in one file, no? `aggregate.ts`